source/vir/src/split_expression.rs

```diff
@@ -498,11 +498,12 @@ pub fn is_split_error(error: &Error) -> bool {
 fn split_call(
     ctx: &Ctx,
     state: &State,
+    diagnostics: &impl Diagnostics,
```

Comment:
is this used for anything in this file? I see it's being passed around mutually recursively, but I can't find a place where it is actually used here.

Reply:
It gets passed to `crate::ast_to_sst::expr_to_exp_as_spec` and `crate::ast_to_sst::expr_to_exp`